src/filter/median.rs

```diff
@@ -242,12 +256,13 @@ impl HistSet {
             data.push([0u32; 256]);
         }
 
+        #[rustfmt::skip]
         HistSet { data, expected_count }
```

Comment:
Sorry to be awkward, but I don't think it makes sense to opt out of the standard formatting for this one line - I was just a bit surprised that rustfmt decided to split this line.

Please could you revert this line and then I'll merge the PR.

Reply:
Okay, I reverted the change.